### DISCUSSION TITLE
Question about Gatsby support

### DISCUSSION THREAD

**Discussion by Isaac-Tait** [gatsbyjs/gatsby#35238] [Community]
A year or so ago I remember reading somewhere on the Gatsby website that if you had an issue that you thought could benefit the Gatsby community at large if solved that you could reach out and schedule a call/chat/email to discuss. The caveat was that if your issue was accepted your session would be video recorded and published to the Gatsby YouTube channel. Is this still an option? If so I have an issue I would like to submit for consideration. How do I begin this process (if Gatsby is still doing it that is)?

**Comment by LekoArts** [comment]
Hi!

This is not something we do anymore. We've updated https://www.gatsbyjs.com/contributing in the meantime to better help in contributions, but if you have specific questions you can ask them here in Discussions or in Discord.

If you have found a bug and want to fix it, you can open an issue (following the template) and say that you want to work on it. Then we can help with figuring out a solution (both in the issue + PR).

Hope that helps, let me know if you have any other questions.

**Reply by Isaac-Tait** [reply]
@LekoArts thank you for your reply here. The issue I am trying to figure out I posted to Discussions [here](https://github.com/gatsbyjs/gatsby/discussions/34689). I recently was contacted by another developer who was trying to figure out the same thing and ended up resorting to manually uploading JSON records to Algolia as they weren't able to figure it out either. I would like to get this sorted out but it is a little out of my realm which is why I was asking for Gatsby support with it. Would this be a better discussion to have with Algolia or do you think Gatsby could help out with it?

**Comment by aminamaiga** [comment]
[StaticQuery: Gatsby : How to download only the data for the corresponding page and avoid sending multiple http requests?]

I am developing a website with gatsbyJs. My site wants to be green (greenIt) with less requests sent to the server. When I inspect the network tab of google chrome, I see that 22 json files were generated on each page. These json files correspond to the different graphQl requests made in the different pages with StaticQuery. My goal is to have one http request sent per page (That is to say, only one json file downloaded for the concerned page).

![gatsby1](https://user-images.githubusercontent.com/35399818/188414251-a3534b62-622b-43ce-84d0-b90e77c26994.PNG)
![gatsby2](https://user-images.githubusercontent.com/35399818/188414260-dcb93dd5-2800-4a9c-8d7b-657838f14316.PNG)
